### ISSUE TITLE
Try codemirror branch

### ISSUE THREAD

**Issue by ivan-kleshnin** [Paqmind/paqmind#4]
Branch introduces uniform code highlighting.

#### Before

1. monospace for inline code
2. highlightjs for block code
3. codemirror for editable code (block only)

#### After

1. codemirror for inline code
2. codemirror for block code
3. codemirror for editable code (block only)

We need to normalize design etc. This branch **is not to merge** but to make separate **codemirror2** branch based on it.

**Comment by scabbiaza** [comment]
Pushed my changes: https://github.com/Paqmind/paqmind/commit/14a6857a853a000a7b29e07b21e31eb3615d53bd

![screen shot 2015-12-31 at 19 26 07](https://cloud.githubusercontent.com/assets/2734464/12066713/295f5752-aff5-11e5-955d-3fdee99d25a3.png)

Some ideas:
1. You did it right: wrapped inline code in `<code>`, wrapped block of code in `<pre><code>`. Here is a bit more information about it: http://stackoverflow.com/questions/4611591/code-vs-pre-vs-samp-for-inline-and-block-code-snippets 
2. I removed code for transformation `div` -> `span`, etc. Much easier is to do with CSS.
3. I've added classes `inline` and `block` for `CodeMirror`, to support first two cases, as you described above. Third case will be without additional class. For example we cannot unset `height: auto` for all cases, or editable CodeMirror will not have scrolls.
4. I've set different paddings for block and inline codes: bigger block requires bigger paddings.
5. I've also cleaned a bit cm-theme.
